environment-api/src/main/java/com/sequenceiq/environment/api/v1/environment/model/response/EnvironmentStatus.java

```diff
@@ -72,7 +72,18 @@
     LOAD_BALANCER_ENV_UPDATE_STARTED("Starting load balancer update for environment"),
     LOAD_BALANCER_ENV_UPDATE_FAILED("Failed to update environment with load balancer"),
     LOAD_BALANCER_STACK_UPDATE_STARTED("Starting load balancer update for all data lakes and data hubs"),
-    LOAD_BALANCER_STACK_UPDATE_FAILED("Failed to update data lakes and data hubs with load balancer");
+    LOAD_BALANCER_STACK_UPDATE_FAILED("Failed to update data lakes and data hubs with load balancer"),
+
+    UPGRADE_CCM_VALIDATION_IN_PROGRESS("Validation for CCM upgrade"),
+    UPGRADE_CCM_VALIDATION_FAILED("Validation for CCM upgrade failed"),
+    UPGRADE_CCM_ON_FREEIPA_IN_PROGRESS("Upgrading CCM on Free IPA"),
+    UPGRADE_CCM_ON_FREEIPA_FAILED("Upgrading CCM on Free IPA failed"),
```

Comment:
Nit: remove space in "Free IPA".

Reply:
there are multiple occurrences with this wording, I correct them too